code/modules/library/lib_machines.dm

```diff
@@ -776,8 +776,13 @@
 	icon_state = "binder"
 	desc = "Only intended for binding paper products."
 	density = TRUE
+
+	/// Are we currently binding a book?
 	var/busy = FALSE
 
+	/// Name of the author for the book, set by scanning your ID.
+	var/scanned_id
```

Comment:
Could be interesting to have that pop up in the examine, or on the contextual clues, rather than just being a `say()` when someone tries to bind a piece of paper before setting themselves as the author with their ID.